### DISCUSSION TITLE
Where to publish an ODF?

### DISCUSSION THREAD

**Discussion by emseedee** [GrandOrgue/grandorgue#881] [Help]
I've just written a simple ODF to use Mark Beverly's Prudhoe Methodist sample set on GO and I thought others may be interested. Where would be a good place to publish it? (I do have a small personal web site, but it runs on an old PC in my study at home, and I'd rather put it somewhere a bit more robust. )

Is there anywhere in the GO Github repository, or could I use my own repository and somehow link it to GO's repository?

Mike

**Comment by larspalo** [comment]
There are many options available.

Of course you can upload to and link from any of your GitHub repositores (you can even create a new special repository for that kind of material so you don't mix contents with other stuff you already have on GitHub). You could also use Google Drive and make a folder public to share with the world. There are many other free web hosts available too, some use Dropbox etc.

We'll gladly link to content you provide for GO!

**Reply by BasKlb** [reply]
Dear Mike,
My suggestion is MPS orgelseite (https://mps-orgelseite.de/home/filebase/). MPS orgelseite offers an excellent possibility to publish GO Odfs, both in German and in English. I have uploaded 18 GO-Odfs so far and have had more than 2000 downloads.

Bas
